libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -20,6 +38,10 @@ $default-checkbox-radio-size: map.get(utils.$checkbox-radio-sizes, 'md');
     --background-checked: #{utils.get-color('success')};
     --border-color-checked: #{utils.get-color('success')};
 
+    @include interaction-state.apply-hover {
+      --background-checked: #{_get-hover-color('success')};
```

Comment:
Should this have loudness xxl or similar too? It is very hard to see the hover effect when it is checked with success color.

Reply:
The request from UX in Zeplin is to use the default loudness. I agree that it's very hard to see, but maybe that discussion should be saved for the UX review?